src/common/options.cc

```diff
@@ -3216,6 +3216,10 @@ std::vector<Option> get_global_options() {
     .set_default(0)
     .set_description(""),
 
+    Option("bluestore_spdk_io_sleep", Option::TYPE_UINT, Option::LEVEL_DEV)
+    .set_default(5)
+    .set_description("Specify the time to sleep when their is no completed I/O"),
```

Comment:
the amount of time to wait before polling for completed SPDK I/O requests.

Reply:
No, your description is not correct. If there is no completed IO, it will wait such time. If every time there are completed I/Os, we will not wait.

Reply:
i reread my comment, its meaning is the same as what your reply.